types/bootstrap/js/dist/button.d.ts

```diff
@@ -11,9 +11,8 @@ declare class Button extends BaseComponent {
      */
     dispose(): void;
```

Comment:
The base component includes the function. Please remove it.

Reply:
amended, thx!

Reply:
cd2bd01e35b4197fdd2f3a89673293ecf7d5bf94